script.js

```diff
@@ -5,19 +5,67 @@ var game = {
   players: [],
   hands: [],
   buildDeck: function(){
+    for (i = 0; i < values.length; i += 1) {
+        for (j = 0; j < suits.length; j += 1)
+    game.deck.push({cardValue: values[i], cardSuit: suits[j]});
```

Comment:
Watch your indentation! Also, looks like you're missing a `{` after your `for()`.